### PR TITLE
Crossentropy to NLL Loss

### LINKED ISSUE #4: A bug of the Loss function in the def 'train' and 'evaluate'
Hi, I found the loss function in the def 'train' and 'evaluate' is cross-entropy. But in the model.py, the output from the decoder is operated by a log-softmax function. According to the definition of cross_entropy, the log_softmax operation has been included in the cross_entropy. I think the loss function in 'train' and 'evaluate' might be nll-loss. Then the entire loss function containing final operation of the decoder and nll_loss is the cross_entropy.

### PR DESCRIPTION
**Pull request by pskrunner14**
Changed crossentropy to nll_loss as crossentropy criterion combines log_softmax and nll_loss in a single function. Fixes #4

### CODE REVIEW

**Comment by keon** [comment]
thanks so much :)
